lib/game_box/players.ex

```diff
@@ -39,6 +39,10 @@ defmodule GameBox.Players do
     GenServer.call(via_tuple(arena_id), {:update_player, player_id, params})
   end
 
+  def register_player(arena_id, player_id, params) do
```

Comment:
this is a slightly different function that will error if the player name is already taken